Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8-slim-buster as base 
+
+RUN pip install poetry
+COPY poetry.lock pyproject.toml /app/todo_app/
+WORKDIR /app/todo_app
+RUN poetry install
```

Comment:
You can use the `--no-dev` flag to skip installing any "dev-dependencies" like pytest.